package.json

```diff
@@ -25,44 +25,44 @@
     "@babel/plugin-proposal-object-rest-spread": "^7.20.7",
     "@babel/preset-env": "^7.26.0",
     "@babel/register": "^7.25.9",
-    "@enonic-types/global": "^7.14.4",
-    "@enonic-types/lib-content": "^7.14.4",
-    "@enonic-types/lib-io": "^7.14.4",
-    "@enonic-types/lib-node": "^7.14.4",
-    "@enonic-types/lib-portal": "^7.14.4",
+    "@enonic-types/global": "^7.15.0-A1",
+    "@enonic-types/lib-content": "^7.15.0-A1",
+    "@enonic-types/lib-io": "^7.15.0-A1",
+    "@enonic-types/lib-node": "^7.15.0-A1",
+    "@enonic-types/lib-portal": "^7.15.0-A1",
+    "@enonic-types/lib-schema": "^7.15.0-A1",
     "@enonic-types/lib-static": "^2.0.0",
     "@enonic/mock-xp": "^1.0.0",
+    "@enonic/react-components": "^6.0.0-B1",
     "@enonic/react4xp": "^5.0.4",
```

Comment:
Shouldn't there be changes (6.0 version) in `@enonic/react-components` as well?

Reply:
?

Reply:
Do you mean @enonic/react4xp? Yeah it looks like we haven't release it with rspack 

https://www.npmjs.com/package/@enonic/react4xp?activeTab=versions 6 months old

Reply:
lib-react4xp doesn't use the build system though, only some constants

Reply:
Upgraded to @enonic/react4xp 6.0.0-B1